typescript/src/accessors/internal_accessor.ts

```diff
@@ -251,6 +254,32 @@ export class InternalAccessor implements IConsumerApiAccessor {
     return this.userTaskService.finishUserTask(identity, processInstanceId, correlationId, userTaskInstanceId, userTaskResult);
   }
 
+  // Tasks
+  public async getAllSuspendedTasks(identity: IIdentity): Promise<DataModels.Tasks.TaskList> {
+    return this.taskService.getAllSuspendedTasks(identity);
```

Comment:
Hier die Referenzen dann auch entsprechend anpassen, z.B:

```suggestion
    return this.flowNodeInstanceService.getAllSuspendedTasks(identity);
```